### PR TITLE
chore(core): add Display bi-directional links setting row ui

### PR DESCRIPTION
**Pull request by JimmFly**
close AF-1269

CleanShot 2024-08-23 at 12 59 50@2x

### CODE REVIEW

**Comment by JimmFly** [comment]
* **#7954** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7954?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @JimmFly and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/components/affine/setting-modal/general-setting/editor/page.tsx
Comment (pengx17) [inline packages/frontend/core/src/components/affine/setting-modal/general-setting/editor/page.tsx:44]: no impl?

Comment (JimmFly) [reply packages/frontend/core/src/components/affine/setting-modal/general-setting/editor/page.tsx:44]: After waiting for #7956  to be merged, the specific functions will be implemented in the new PR.

**Review by pengx17 — APPROVED** [review]

**Pull request merged**
